build.cmd

```diff
@@ -19,16 +19,18 @@ if not exist %NUGET_CACHE_DIR%\NuGet.exe (
 )
 
 rem copy nuget locally
-if not exist .nuget\NuGet.exe (
-  if not exist .nuget md .nuget
-  copy %NUGET_CACHE_DIR%\NuGet.exe .nuget\NuGet.exe > nul
-)
+if not exist .nuget md .nuget
+copy %NUGET_CACHE_DIR%\NuGet.exe .nuget\NuGet.exe > nul
 
 rem restore packages for build script
 .nuget\NuGet.exe restore .\packages.config -PackagesDirectory .\packages -Verbosity quiet
 
+rem find Visual Studio and MSBuild installation paths
+FOR /F "delims=" %%i IN ('.\packages\vswhere.1.0.62\tools\vswhere.exe -property installationPath') DO set VS_PATH=%%i

```

Comment:
>  Csi.exe is included in the Microsoft.Net.Compilers package, so we could use this instead of looking for csi.exe in the VS2017 installed.

It _is_ much cleaner, but I'm a little torn. We're pulling down 22.5 MB that we likely don't need. (Only once, I suppose.)